src/main/app/claims/claimStoreClient.ts

```diff
@@ -6,21 +6,31 @@ import { ClaimModelConverter } from 'claims/claimModelConverter'
 import { ForbiddenError } from '../../errors'
 import { Draft } from '@hmcts/draft-store-client'
 import { DraftLegalClaim } from 'drafts/models/draftLegalClaim'
+import * as toBoolean from 'to-boolean'
 
 const claimApiBaseUrl = `${config.get<string>('claim-store.url')}`
 const claimStoreApiUrl = `${claimApiBaseUrl}/claims`
 
 export default class ClaimStoreClient {
   static saveClaimForUser (user: User, draft: Draft<DraftLegalClaim>): Promise<Claim> {
     const convertedDraftClaim: object = ClaimModelConverter.convert(draft.document)
-    return request.post(`${claimStoreApiUrl}/${user.id}`, {
+
+    return request.post(this.getUri(user), {
       body: convertedDraftClaim,
       headers: {
         Authorization: `Bearer ${user.bearerToken}`
       }
     })
   }
 
+  static getUri (user: User) {
+    if (toBoolean(config.get('featureToggles.inversionOfControl'))) {
+      return `${claimStoreApiUrl}/${user.id}/create-legal-rep-claim`
```

Comment:
we could've passed the user id directly, but 👍

Reply:
👍